pages/templates/pages/index.html

```diff
@@ -64,19 +64,19 @@ <h3 class="title text-lightest">FAQ</h3>
     </div>
     <div class="row">
       <div class="col-xs-12  col-sm-6">
-        <div class="faq__box  faq__box--big" tabindex="1">
+        <div class="faq__box  " tabindex="1">
```

Comment:
all these `class` atributes have extra whitespace at the end, can you please remove.